recipes/BEHST/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+mkdir $PREFIX/../results/
+
+mkdir $PREFIX/../temp/
+
+
+sh project.sh $PREFIX/../data/pressto_LUNG_enhancers.bed DEFAULT_EQ DEFAULT_ET
```

Comment:
You can selectively copy only executables to $PREFIX. That [chromhmm recipe](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/chromhmm) I mentioned does that -- it ignores the data directory, only copies over the jar file, and adds a script for downloading the data later.